internal/fsutils/fsutils.go

```diff
@@ -91,3 +92,44 @@ func EvalSymlinks(path string) (string, error) {
 
 	return er.path, er.err
 }
+
+// code from https://github.com/guillermo/doubleglob/blob/main/double_glob.go
+var replaces = regexp.MustCompile(`(\.)|(\*\*/)|(\*)|([^/\*.]+)|(/)`)
+
+func toRegexp(pattern string) string {
+	pat := replaces.ReplaceAllStringFunc(pattern, func(s string) string {
+		switch s {
+		case "/":
+			return "\\/"
+		case ".":
+			return "\\."
+		case "**/":
+			return ".*"
+		case "*":
+			return "[^/]*"
+		default:
+			return s
+		}
+	})
+	return "^" + pat + "$"
+}
+
+func FileNameMatchMask(name, pattern string) bool {
```

Comment:
Why is this a global function? Will it be used elsewhere?

Reply:
Yes. I assume that this function can be used in the future.